gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ androidTargetSdk = "32"
 androidx-lifecycle = "2.4.1"
 androidx-navigation-compose = "2.5.1"
 coil = "2.2.1"
-compose = "1.2.1"
+compose = "1.3.0"
```

Comment:
Update the compiler as well?

Reply:
it's tied to kotlin version which I want to keep aligned with the main project

Reply:
👍